### DISCUSSION TITLE
Submit your templates!

### DISCUSSION THREAD

**Discussion by SilentVoid13** [SilentVoid13/Templater#138] [Templates Showcase]
You can now submit your own templates made with Templater here. Just create a new discussion under the "Templates Showcase" category.
Name the new discussion wisely, so that we can quickly understand what the template does.

Use **gists** to upload your template file: https://gist.github.com/. This has multiple advantages:
- This enables version control for your file, if you want to update and enhance it
- You can share the link everywhere (obsidian forums, Discord, ...) and everybody will be able to see the same updated file

Can't wait to see what the community has come up with using Templater!

**Comment by QuaCKeReD** [comment]
Here is my first real foray into templater 😀 I hope this helps someone.

The reasoning behind this workflow is;
- I wanted to use YAML front matter for data view attributes - having them in the rendered file doesnt work for me, as I transclude lots of these to send as a report.
- I work for a company that has clients, each client can have several customers and each customer can have a variety of projects I am dealing with. 
- I also want to be able to create notes for personal reviews/research/training/etc.

The new note.md file is used as my default new file - templater is set to listen for new file creation

1. It reads a list of options from the noteYamlOptions.md file - these entries are comma separated combinations for my common area/company/client/project options
2. Next, it splits this csv string into entries to use in the front matter.
3. After that, it then prompts for a selection from the noteYamlTypes.md file - this is as I can have many types of file against any of the combinations from options
4. Finally, it renames the file and moves it to my diary folder structure

[Gist](https://gist.github.com/QuaCKeReD/0c0348c6decf60ec5a0a069c0d83a990)

**Comment by tiagozip** [comment]
Obsidian AI, a template to run GPT3, GPT4 and LLAMA (fallback) into Obsidian AI for free!

* No API key needed (uses reverse-engineered just like gpt4free)
* Uses native selector/prompter
* Rewrite
* Expand text
* Chat directly
* Add your custom actions
* And much more!

Link: https://github.com/SilentVoid13/Templater/discussions/1105
